src/App.tsx

```diff
@@ -13,190 +41,36 @@ export const App: React.FC = () => {
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this button should have `active` class only if all todos are completed */}
-          <button
-            type="button"
-            className="todoapp__toggle-all active"
-            data-cy="ToggleAllButton"
-          />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              data-cy="NewTodoField"
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+        <Header />
+        {todos.length > 0 && (
```

Comment:
```suggestion
        {!!todos.length && (
```

Reply:
So, can you explain me the advantage of '!!' comparing to '>0' in this situation or in general?

Reply:
So, can you explain me the advantage of '!!' comparing to '>0' in this situation or in all?